LukasHH/mod_itcs_reservation_calendar
# LukasHH/issue3-1 - Second Version to Fixes #3

This is a second suggestion to solve the problem with time zones (Fixes #3).
In this version I created a separate form field to select the date. It is based on the HTML5 element <input type="date">. The calendar in the form is provided by the browser and may look and handle differently depending on the browser used. In the database only the date is stored and no more times.

When updating from version 4.0.3 to 4.0.4-alpha1 the existing data will be changed to the new format.

To test:
1. install the version 4.0.3 in a test version.
2. set a time zone in the Joomla configuration, which is above the UTC time. (e.g. Berlin +2h)
3. open the module in the backend, choose a date (e.g. 05.09.2022) and save the module.
4. under Joomla 3
a.	In the database we save the date with UTC time (starting from 0 o'clock) under params. (e.g. 04.09.2022 22:00:00)
b.	In the module the 04.09.2022 is shown in the calendar.
5. under Joomla 4
a.	In the database we store the date with UTC time (starting from the current time) under params. (e.g. "05.09.2022 07:00:00" if saved at 09:00 o'clock).
b.	In the module, the calendar will display 05.09.2022.
c.	If the module is saved at 1 a.m., the date in the database changes (e.g. to 04.09.2022 23:00:00)
6. perform the update to version 4.0.4-alpha1 (https://github.com/LukasHH/mod_itcs_reservation_calendar/releases/tag/4.0.4-alpha1)
7. the date is displayed correctly in the frontend under Joomla 4 and Joomla 3.
